src/send/SendEnterAmount.tsx

```diff
@@ -24,7 +24,12 @@ const TAG = 'SendEnterAmount'
 
 function SendEnterAmount({ route }: Props) {
   const { defaultTokenIdOverride, origin, recipient, isFromScan, forceTokenId } = route.params
-  const tokens = useTokensForSend()
```

Comment:
@kathaypacific Sorry, I missed this PR.
But, I have a little concern with this...
I think we should be consistent, and if we change the logic of what tokens to show in the send flow, that should be inside `useTokensForSend`?
`useTokensForSend` is used in the TokenDetails to decide if we show the "Send" quick action, if the token appears in the send flow, I would expect to see the "Send" button, right?